ScintillaNet_FindReplaceDialog/GoTo/GoToDialog.cs

```diff
@@ -139,9 +140,10 @@ public Scintilla Scintilla
 
         #region Constructors
 
-        public GoToDialog()
+        public GoToDialog(int startingLineIndex)
```

Comment:
The constructor should be overloaded to keep backwards compatibly in check.

Reply:
I've done this, but again I'm new to git and GitHub so I'm not sure if I've added correctly.

Reply:
This looks good now.